libs/utils/trace.py

```diff
@@ -638,11 +647,12 @@ def _sanitize_SchedEnergyDiff(self):
             ['< 150', '< 400', '< 600'], '>= 600')
 
         # Tag columns by nrg_payoff
-        ccol = df.nrg_payoff
-        df['nrg_payoff_group'] = np.select(
-            [ccol > 2e9, ccol > 0, ccol > -2e9],
-            ['Optimal Accept', 'SchedTune Accept', 'SchedTune Reject'],
-            'Suboptimal Reject')
+        if 'nrg_payoff' in df:
```

Comment:
Can `nrg_payoff` really be missing?
It should be available on both the old and new format, isn't it?

In any case, if we don't add the `nrg_payoff_group` columns, the plot for the "PE filtering" space will be broken.